xm-commons-domainevent/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     implementation project(':xm-commons-lep')
 
     implementation "org.springframework.boot:spring-boot-starter-web:${versions.springBoot}"
+    implementation "org.hibernate:hibernate-core"
```

Comment:
Need to consider: use this module or move DataBaseSource interceptor changes to separate module like: xm-commons-domainevent-database

Reply:
Replied at xm chat: move this changes to com.icthh.xm.commons.domainevent.db